src/constructs/iam/policies/s3-get-object.test.ts

```diff
@@ -46,3 +46,35 @@ describe("The GuGetS3ObjectPolicy class", () => {
     });
   });
 });
+
+describe("The GuGetDistributablePolicy construct", () => {
```

Comment:
Is it worth testing that the correct parameter is added too?

Reply:
Yep! Looks like this can only be done via snapshot testing as there's no `toHaveParameter` assertion available.

Reply:
Done.

Reply:
Moving this comment here as I managed to post it on the wrong thread the first time 🤦 

I've done something along the lines of this elsewhere which might work here too

```ts
const json = SynthUtils.toCloudFormation(stack) as SynthedStack;

expect(json.Parameters.DistributionBucketName).toEqual({
  Default: "/account/services/artifact.bucket",
  Description: "SSM parameter containing the S3 bucket name holding distribution artifacts",
  NoEcho: true,
  Type: "AWS::SSM::Parameter::Value<String>",
});
```

Reply:
Have implemented this style of test now.